### PR TITLE
[KAG-1585] Convert install pages to use Cloudsmith

### PR DESCRIPTION
**Pull request by curiositycasualty**
### Description

Update all installation instructions to use Cloudsmith rather than Pulp.
This PR should not be merged until the blog post regarding these changes is published.

This re-introduces the changes from https://github.com/Kong/docs.konghq.com/pull/5763 which were reverted pending the publishing of the associated blog post.

### Testing instructions

Netlify link:
- https://deploy-preview-5856--kongdocs.netlify.app/
- https://deploy-preview-5856--kongdocs.netlify.app/gateway/3.3.x/install/linux/ubuntu/#advanced-installation

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: tools/install-tester/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Install Tester
+
+This is a tool that scrapes Kong's installation documentation and uses `docker` to ensure that the packages install as expected.
+
+## Why does it exist?
+
+Kong's install docs should always work. That's where everyone starts on the docs.
+
+Why now? A tool was needed to validate install docs while moving to a new package hosting platform (Pulp to Cloudsmith):
```
[the remark this answers is not in the document]
Comment (curiositycasualty) [reply tools/install-tester/README.md:9]: Resolved by adding "Cloudsmith" to Vale vocab file.

**Comment by curiositycasualty** [comment]
This is ready for review, but it needs to wait on the blog post being published to be merged.

**Comment by curiositycasualty** [comment]
Blog post is live: https://konghq.com/blog/product-releases/changes-to-kong-package-hosting

**Comment by mheap** [comment]
@curiositycasualty The lsb-release change was made intentionally to make all tests pass. Please revert your changes for now and we can discuss switching to lsb-release when I’m back

**Comment by mheap** [comment]
I recommend resetting to 8b60719c38e36f8278c96b473ed0d9baa0b6e073 which was previously merged.

We can do the Vale changes later too

**Comment by curiositycasualty** [comment]
Closing in favor of https://github.com/Kong/docs.konghq.com/pull/5859

**Pull request closed without merging**